js/index.js

```diff
@@ -4,16 +4,21 @@
  * @flow
```

Comment:
Please remove this comment "https://github.com/facebook/react-native" or replace it with GitHub link for of your project.